src/main/java/mekanism/common/FuelHandler.java

```diff
@@ -56,7 +58,7 @@ public FuelGas(int duration, double energyDensity)
 		public FuelGas(IFuel bcFuel)
 		{
 			burnTicks = bcFuel.getTotalBurningTime() / Fluid.BUCKET_VOLUME;
-			energyPerTick = bcFuel.getPowerPerCycle() * general.FROM_RF;
+			energyPerTick = bcFuel.getPowerPerCycle() * MjAPI.MJ;
```

Comment:
Thanks for looking at this!

What I see from this, is a cross mod compatability feature allowing BC fuel burn in the Mekanism GasGenerator. It's about adding features to Mekanism, not to BC ;)

The desired result is energyPerTick in Joules as that's what Mekanism uses internally if I'm right. 
With 1J being: 10EU, 2.5RF (and Tesla, ForgeEnergy).

If I'm reading the JavaDoc right, getPowerPerCycle returns micro mj, so multiplying with MjAPI.MJ (1.000.000) should return Joule right?